### DISCUSSION TITLE
posts/blog-building/

### DISCUSSION THREAD

**Discussion by giscus** [CattoFace/barrcodes#5] [Announcements]
# posts/blog-building/

My “Hello World” post about how I made this blog with no frameworks and client-side dependencies.

https://barrcodes.dev/posts/blog-building/

**Comment by Zlkva** [comment]
Now that the site has been updated to Hugo, how many lines of code are you up to now?

**Reply by barr-israel** [reply]
77 lines! but Hugo actually didn't affect the size at all.
Changing fetch a little to also allow changing meta tags and switching to `color-scheme` theming added exactly 2 line to the total,  
And **19** lines purely dedicated to prevent the "fetch and replace" from breaking the comments section.
